### ISSUE TITLE
Suggestion: Use art submitted by community for per-release splash arts

### ISSUE THREAD

**Issue by tiritto** [Orama-Interactive/Pixelorama#59]
This is rather long-term suggestion due to Pixelorama being in early phase of development.

It could be a nice idea to incorporate Pixelorama (at some point hopefully huge) community into creation of its splash arts. Some people might be already familiar with such concept as I'm pretty sure that Photoshop does this with their splash arts too.

Basically, members of our community could submit creations they made with Pixelorama and the best looking art would become Pixelorama splash art for next minor release. On top of that, authors name would be displayed on the splash art itself and within "About" information page, possibly with some link to their portfolio or project page. Then once time comes for next release, the cycle would continue with another Pixelorama artwork from another artist.

The only issue is that Pixelorama must follow MIT license, meaning that art itself would have to be published on MIT license as well. However, I feel like it wouldn't discourage as many people as it might feel like. Moreover, since there is no way to get around this issue we could just embrace all it's aspects by publishing artwork project file alongside current release. This way Pixelorama users could not only see what kind of amazing art can be made (like the one on splash art) but also peek inside splash art project to see how it was made in the first place. You could access current splash art project file from within "About" section next to author credits.

If Pixelorama would manage to reach the point of being capable of hosting own website, such approach could even transcend into some kind of "Hall of Fame" in the future, where all splash arts would be published alongside creator information and project files.

**Comment by Calinou** [comment]
> The only issue is that Pixelorama must follow MIT license, meaning that art itself would have to be published on MIT license as well.

This actually isn't required. However, it's a good idea to require submitted artwork to be under some kind of free license (CC0, CC BY or CC BY-SA). Of course, credit has to be given on the splash itself with a small watermark.

**Comment by OverloadedOrama** [comment]
I love this idea! I think it would be better if it's done the same way Blender does it, instead of putting art in the splash screen, because Pixelorama loads too fast. And if in some point in the future it takes more time to load, we could change it and let it be in the splash screen instead.

![image](https://user-images.githubusercontent.com/35376950/71032609-59599500-211e-11ea-9a62-f32b72564e2d.png)

**Comment by OverloadedOrama** [comment]
Hey, update on the situation. We implemented a splash screen on v0.6, and we made it so Patrons, from the Visionary tier and onward, can participate in an artwork contest for Pixelorama's splash screen. The art will be updated in every major version (v0.7, then v0.8, ..., then v1.0, then v1.1 and so on). I'll go ahead and close this issue as the suggestion has been implement, but if you'd like to suggest changes, feel free to reply here! Thank you!
